scripts/tail-begin-block-events.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+const WebSocket = require('ws');
+const util = require('util');
```

Comment:
I like this file as an example. 

It would help to add a comment here as how this file would typically be used. like
```shell
# using Node.js: setup a WebSocker listener
node ./scripts/tail-begin-block-events.js
# run a fast chain test
sh ./scripts/incentives-setup.sh
```

Reply:
Feel free to add to the PR. I think it's pretty self-explanatory how to run shell scripts.

Reply:
Fair enough, it is a pretty simple script